pkg/controller/extension/actuator.go

```diff
@@ -75,6 +76,10 @@ func (a *actuator) Reconcile(ctx context.Context, log logr.Logger, ext *extensio
 		return fmt.Errorf("error creating shoot client: %w", err)
 	}
 
+	if err := ReconcileShootInfoConfigMap(ctx, log, shootClient, config, cluster); err != nil {
```

Comment:
We should probably delete the `ConfigMap` on extension deletion, i.e., in `actuator.Delete()`.

Reply:
I did not add a delete for this resource because we don't have it for the other provisioned resources either. Why would we want to delete this configMap when the user should clean up the rest if he want's to remove flux?

Reply:
I thought that leaving an orphaned shoot info ConfigMap with potentially outdated data in the system could lead to confusion or even problems.
But removing it even if it's still in use could also be harmful.
Let's keep it on deletion then and document that the ConfigMap will be left in the shoot and will no longer be updated.